md/release-notes.md

```diff
@@ -4,4 +4,49 @@
 **Note:** The 7.13 is currently work in progress (WIP). The 7.13.0 GA is planned on June 2021.
 :::
 
+## Features
 
+::: info
+TODO
+:::
+
+## Improvements
+
+### Development suite changes
+
+### Runtime changes
+
+#### Groovy script results casting
+
+Bonita runtime will automatically try to cast result of groovy script when it strictly differ from the declared return type.
```

Comment:
```suggestion
Bonita runtime will automatically try to cast result of groovy script when it differs from the declared return type but when it can be converted.
```